tests/functions.sh

```diff
@@ -64,14 +71,49 @@ function ssh_no_ping {
     fi
 }
 
+function ssh_udp {
+    id1=$1
+    id2=$2
+    port=$3
+    ssh_run_background $id1 "nc -lup $port > /tmp/test"
+    ssh_run_background $id2 "echo 'this message is from vm $id2' | nc -u -w 1 42.0.0.$id1 $port"
+    ssh_run $id2 "sleep 1"
+    ssh_run $id1 "cat /tmp/test"
```

Comment:
is this cat still usefull ?
if the goal is to test the existence of the file, it's done with [ -s "/tmp/test" ]